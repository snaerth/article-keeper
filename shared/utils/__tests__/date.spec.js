import formatISODateTime, { checkTime } from '../date';

describe('Run tests for date', () => {
  it('Check if time is bigger or smaller than zero', () => {
    const result = checkTime('9');
    expect(result).toMatch('09');
    expect(result).not.toBe('9');
  });

  it('Format date string to ISO date time format', () => {
    const success = formatISODateTime('2017-08-11T12:41:59.791Z');
    expect(success).toMatch('2017-08-11 12:41:59');
    const fail = formatISODateTime('test');
    expect(fail).toThrowErrorMatchingSnapshot();
  });
});
