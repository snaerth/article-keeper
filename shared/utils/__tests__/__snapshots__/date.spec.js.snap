// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Run tests for date Format date string to ISO date time format 1`] = `"received is not a function"`;
